### DISCUSSION TITLE
how to report the value of overflow after routing?

### DISCUSSION THREAD

**Discussion by imcamino** [The-OpenROAD-Project/OpenROAD-flow-scripts#961] [General] [answered]
how to report the value of overflow after routing?

**Comment by vijayank88** [comment, marked as answer]
It was already part of the flow [script](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/83c5f697b93b8e8a7165679c748ec26b98e8ed2c/flow/scripts/global_route.tcl#L20)
Just check in your `reports/platform/design/base/congestion.rpt`

**Reply by imcamino** [reply]
Thanks for your answer
